bin/foreman-node

```diff
@@ -15,6 +15,10 @@ require 'net/https'
 require 'etc'
 require 'timeout'
 
+if SETTINGS[:filecache]
+  require 'msgpack'
```

Comment:
This means, we should package msgpack in foreman-packaging.

Reply:
Hope it's ok for you, I'm trying to add msgpack to foreman-packaging:
https://github.com/theforeman/foreman-packaging/pull/5520

Reply:
Nice, sure. This wasn't a big problem for us as we are running it containerized and we can run gem install during image building but in a "real" OS installing things bypassing the package managed is a pain indeed. Thanks!